### PR TITLE
[#2608] improvement(IT): Splitting WEB-E2E IT into an alone Github Action

### PR DESCRIPTION
**Pull request by xunliu**
### What changes were proposed in this pull request?

1. Add Gradle compile parameter `skipWebITs`
2. Splitting integration-test Github Action

### Why are the changes needed?

Run WEB E2E integration test in alone Github Action.

Fix: #2608 

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

CI passed

### CODE REVIEW

File: .github/workflows/frontend-integration-test.yml
Comment (yuqi1129) [inline .github/workflows/frontend-integration-test.yml:98]: For the front-end test, this line can be removed.

File: integration-test/build.gradle.kts
Comment (mchades) [inline integration-test/build.gradle.kts:194]: Why remove this line?

Comment (xunliu) [reply integration-test/build.gradle.kts:194]: Because it duplicated, It's already exist in https://github.com/datastrato/gravitino/blob/main/build.gradle.kts#L163

File: .github/workflows/frontend-integration-test.yml
Comment (xunliu) [reply .github/workflows/frontend-integration-test.yml:98]: Maybe we need keep retain these, Because front-end test also need running in the `Gravitino Server deply mode` and `embedded mode`
```
            integration-test/build/integration-test.log
            distribution/package/logs/gravitino-server.out
            distribution/package/logs/gravitino-server.log
```
What you think?

Sure, I need remove `catalogs/**/*.log`.

Comment (yuqi1129) [reply .github/workflows/frontend-integration-test.yml:98]: Okay,that is exactly what I mean.

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Comment by jerryshao** [comment]
What is the difference compared to the previous CI workflow? Separating two workflows doesn't help the CI, the CI will still fail if web e2e test is not stable, and we still cannot merge it, we can only merge the PR when all the CI checks pass, am I right?

**Comment by xunliu** [comment]
> What is the difference compared to the previous CI workflow? Separating two workflows doesn't help the CI, the CI will still fail if web e2e test is not stable, and we still cannot merge it, we can only merge the PR when all the CI checks pass, am I right?

Splitting two difference CI would help a lot:
1. When we commit web PR, We can execute this Web-E2E test CI many times in isolation. Especially in the case of unstable web tests, we will require that each web test submission needs to be executed successfully multiple times in a row before merging the
2. It can easy to help our found Web E2E failed logs. 
3. when backend engineer modify core or API, maybe will trigger web issue, backend Action success, but frontend Action failed. in the case backend engineer would easy found issue and to request frontend engineer help

**Pull request merged**
